workflows/diagnostics/tests/prognostic/emulation/test_single_run.py

```diff
@@ -7,217 +7,217 @@
 cdl = """
 netcdf out {
 dimensions:
-	time = 3;
```

Comment:
changed from tabs to spaces. vscode was having trouble telling what the indentation type should be.